dist/angular-eonasdan-datetimepicker.js

```diff
@@ -25,12 +25,12 @@
                     }, true);
 
                     ngModel.$render = function () {
-                        if (!ngModel.$viewValue) {
-                            ngModel.$setViewValue(null);
-                        } else if (!moment.isMoment(ngModel.$viewValue)) {
+                        if (!_.isUndefined(ngModel.$viewValue) && !moment.isMoment(ngModel.$viewValue)) {
```

Comment:
Since this project does not require `lodash` I would stick to `ngModel.$viewValue !== undefined`